ods_ci/libs/Helpers.py

```diff
@@ -99,7 +99,7 @@ def update_notification_email_address(self, cluster_name, email_address, addon_n
     @keyword
     def convert_to_hours_and_minutes(self, seconds):
         """Converts seconds in hours and minutes"""
-        m, s = divmod(int(seconds), 60)
+        m, _s = divmod(int(seconds), 60)
```

Comment:
Personally, I would slightly prefer to have just `_` in all these variables you renamed (this way it is more visible for myself that we're ignoring this variable), but I don't really insist.

Reply:
This is related to the previous comment. I wanted to preserve the variable name "for documentation purposes". I'll go and put just a `_` there; that's probably how this is most often done.